main.py

```diff
@@ -0,0 +1,15 @@
+
+
+N = input("Enter a number: ")
+deck = set()
+score_list = list()
+for i in range(int(N)):
+    deck.add(str(i+1))
+for i in range(int(N)):
+    score_list.append(int(deck.pop()))
+    print("Player " + str(i+1) + " draw " + str(score_list[i]))
+print("Player " + str(score_list.index(max(score_list))+1) + " is the winner with " + str(max(score_list)) + " points")
```

Comment:
You assume the number of the players equal to the number of cards. 
Let's make it more interesting. 
40 card. and n players (n between 2-6)

Reply:
use f formating

Reply:
print("Player " + str(score_list.index(max(score_list))+1) + " is the winner with " + str(max(score_list)) + " points")
try to the the prases here  str(score_list.index(max(score_list))+1)  and  str(max(score_list))  and put them into variables so it would be eaiser to understand what they doing.

Reply:
do that for all the complex logic inside a strings